proposals/12147-mlmd-removal/kep-mlmd-database-migration.md

```
@@ -0,0 +1,471 @@
+# KEP: MLMD to KFP Native Database Migration
+
+## Table of Contents
+
+1. [Summary](#summary)
+2. [Motivation](#motivation)
+3. [Goals](#goals)
+4. [Non-Goals](#non-goals)
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)